docs/pages/project/community.md

```diff
@@ -18,7 +18,7 @@ The Meshery project is community-built and welcomes collaboration!
 <br /><a href="https://github.com/meshery/meshery"><i class="fab fa-github"></i> Fork here on Github</a>
 
 * Join any of the [community meetings](https://meshery.io/calendar). 
-  * Watch [meeting recordings](https://www.youtube.com/@Layer5io?sub_confirmation=1) and subscribe to the [community calendar](https://meshery.io/calendar).
+* Watch [meeting recordings](https://www.youtube.com/@Layer5io?sub_confirmation=1) and subscribe to the [community calendar](https://meshery.io/calendar).
```

Comment:
this indentation is intentional. please revert.

Reply:
Done, @leecalcote, PTAL, Thanks!